tests/components/litterrobot/test_button.py

```diff
@@ -31,7 +27,7 @@ async def test_button(hass: HomeAssistant, mock_account: MagicMock) -> None:
 
     entry = entity_registry.async_get(BUTTON_ENTITY)
     assert entry
-    assert entry.entity_category == ENTITY_CATEGORY_CONFIG
+    assert entry.entity_category == EntityCategory.CONFIG
```

Comment:
Please use `is` for `entity.entity_category` comparison

Reply:
Agreed, done.